src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -6680,23 +6693,26 @@ private StatementSyntax ParseStatementNoDeclaration(bool allowAnyExpression)
                     {
                         return this.ParseExpressionStatement(this.ParseQueryExpression(0));
                     }
-                    else
-                    {
-                        goto default;
-                    }
+                    break;
+            }
 
-                default:
-                    if (this.IsPossibleLocalDeclarationStatement(allowAnyExpression))
-                    {
-                        return null;
-                    }
-                    else
-                    {
-                        return this.ParseExpressionStatement();
-                    }
+            if (this.IsPossibleLocalDeclarationStatement(allowAnyExpression))
+            {
+                return null;
+            }
+            else
+            {
+                return this.ParseExpressionStatement();
             }
         }
 
+        private SyntaxToken ParseAwaitKeywordForAsyncStreams()
+        {
+            Debug.Assert(this.CurrentToken.ContextualKind == SyntaxKind.AwaitKeyword);
+            SyntaxToken awaitToken = this.EatContextualToken(SyntaxKind.AwaitKeyword);
+            return CheckFeatureAvailability(awaitToken, MessageID.IDS_FeatureAsyncStreams);
```

Comment:
Are we grouping `await using` as part of async streams now?

Reply:
That's always been the case. There is a single feature with four parts: async using, async foreach, async iterators and async LINQ.

Reply:
seems *super* weird.  I get the relation between foreach/iterators and linq.  I even get that we might want to do all this work at the same time.  but calling 'async using' some sort of 'async stream' feature just doesn't make sense to me :)

That said, it seems minor.  It's just confusing at times to read things this way.